src/pytest_workflow/plugin.py

```diff
@@ -481,7 +490,18 @@ def runtest(self):
         assert self.workflow.exit_code == self.desired_exit_code
 
     def repr_failure(self, excinfo, style=None):
-        message = (f"'{self.workflow.name}' exited with exit code " +
-                   f"'{self.workflow.exit_code}' instead of "
-                   f"'{self.desired_exit_code}'.")
+        standerr = self.workflow.stderr_file
+        standout = self.workflow.stdout_file
+        with open(standout, "rb") as standout_file, \
+             open(standerr, "rb") as standerr_file:
+            if os.path.getsize(standerr) >= self.standerderror_bytes:
+                standerr_file.seek(-self.standerderror_bytes, 2)
```

Comment:
`2` is niet heel erg informatieef. Er is een named constant in de `io` module die je kunt gebruiken.